src/commands/calc.rs

```diff
@@ -49,9 +49,15 @@ fn calc(input: Value, args: &RawCommandArgs) -> Result<OutputStream, ShellError>
 }
 
 pub fn parse(math_expression: &str, tag: impl Into<Tag>) -> Result<Value, String> {
+    use std::f64;
     let num = meval::eval_str(math_expression);
     match num {
-        Ok(num) => Ok(UntaggedValue::from(Primitive::from(num)).into_value(tag)),
+        Ok(num) => {
+            if num == f64::INFINITY || num == f64::NEG_INFINITY {
+                return Err(String::from("Cannot divide by zero"));
```

Comment:
Since infinity could be produced in other ways (e.g., `log(0)`), how about we have a more generic error message like `"cannot represent result"`?

On a side note, we should think about whether or not we want to be able to represent these 🙂